schedview/app/scheduler_dashboard/scheduler_dashboard.py

```diff
@@ -127,7 +127,7 @@ class Scheduler(param.Parameterized):
     )
     nside = param.ObjectSelector(
         default=16,
-        objects=[2**n for n in np.arange(1, 6)],
+        objects=[2, 4, 16, 32, 64],
```

Comment:
The dashboard currently displays nside options [2, 4, 8, 16, 32].
@ehneilsen can you clarify which options should be given?

Reply:
These are fine. Mostly I think people will want 16, 32, or 64, or maybe 8 if the user is impatient or testing. I doubt 2 or 4 will get used much, but there presence is harmless. At present, higher than 64 is not useful, because none of the instances of scheduler we have right now use nside>64. I suppose its possible that someday there might be data with higher nsides than this, but I don't think we should worry about adding them for something so speculative.
If you wanted to be really fancy, you could offer the powers of 2 up to and including `scheduler.nside`, where `scheduler` is whatever instance of the scheduler is loaded, but really I don't think it's worth the effort to make it dynamic like this.
`nside` is _always_ a power of 2.